### DISCUSSION TITLE
Set default timezone for serverless functions

### DISCUSSION THREAD

**Discussion by raikusy** [vercel/vercel#4158] [General] [answered]
For node.js applications, I can set a environment variable like `TZ=Asia/Dhaka` to set the default timezone. So that dates in my app and database remains synced with my local time.

While I try to add TZ env in now project settings, it says this variable is reserverved.

How can I set the default timezone for my serverless function?

**Comment by styfle** [comment, marked as answer]
You might be able to run `process.env.TZ = 'Asia/Dhaka'` as the first line of your serverless function, but I think it is best to avoid relying on timezones in your functions and instead use [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time) (which is the default `TZ` value).

**Reply by raikusy** [reply]
Thanks!

**Reply by MunnyReol** [reply]
Thanks

**Reply by vyrazuarkaprabha** [reply]
In where I have to call it?

**Reply by vyrazuarkaprabha** [reply]
please hare me your yml file

**Reply by MunnyReol** [reply]
> In where I have to call it?

Use this instead of that 

``new Date().toLocaleString("en-US", {timeZone: "Asia/Kolkata"})``

**Reply by jinan159** [reply]
Check this out. 'TZ' is defined environment variable on **AWS Lambda**.

[https://vercel.com/docs/environment-variables#reserved-environment-variables](https://vercel.com/docs/environment-variables#reserved-environment-variables)
